include/tunable_reduction.h

```diff
@@ -323,7 +323,7 @@ namespace quda
     {
       bool rtn;
       do {
-        rtn = Tunable::advanceBlockDim(param);
+        rtn = TunableReduction2D::advanceBlockDim(param);
```

Comment:
I think we may have discussed this, but did this unexpected prior implementation of calling `Tunable::` methods instead of `TunableReduction2D::` cause issue somewhere?

Reply:
Yes, the issue was in the gauge loop trace routine, where I tuned over a 3-d space (`x_cb`, parity, and loop id) and wanted to reduce only over the first two dimensions. This is the first time such a tuning space + reduction was introduced in the code, which is why it didn't get hit in the past. (The only other place a close pattern gets hit is the multi-reduction, which is a 2-d space of NXZ and NYW.)